lib/manager.rb

```diff
@@ -0,0 +1,77 @@
+# Require gems
+require "date"
+
+# Require relatives
+require_relative "room.rb"
+require_relative "reservation.rb"
+
+module Hotel
+  class Manager
+    attr_reader :rooms, :reservations
+
+    def initialize
+      @rooms = (1..20).map do |room_number|
+        Hotel::Room.new(room_number: room_number)
+      end
```

Comment:
Since we're inside the `Hotel` module already, you don't need the `Hotel::` here.